securedrop/journalist_app/col.py

```diff
@@ -91,6 +93,15 @@ def download_single_file(filesystem_id, fn):
         if '..' in fn or fn.startswith('/'):
             abort(404)
 
+        file_ = current_app.storage.path(filesystem_id, fn)
+        if not Path(file_).is_file():
```

Comment:
[nit] it's ok to use file as a variable name, see how it's no longer a builtin: https://docs.python.org/3/library/functions.html#built-in-functions